sources/packages/api/src-sql/MSFAANumbers/Create-msfaa-numbers-table.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE IF NOT EXISTS sims.msfaa_numbers (
+  id SERIAL PRIMARY KEY,
+  msfaa_number BIGINT NOT NULL,
+  date_requested DATE,
```

Comment:
is this only Date or a Datetime object

Reply:
Changed as below following the same done for sims.applications.application_status_updated_on
```
date_requested TIMESTAMP WITH TIME ZONE
```